service/metrics/init.go

```
@@ -10,7 +10,8 @@ import (
 var registry *prometheus.Registry
 var once sync.Once
 var counterHandler map[counterMetrics]*prometheus.CounterVec
-var histogramHandler map[string]*prometheus.HistogramVec
+var histogramHandler map[counterMetrics]*prometheus.HistogramVec
```

golangci-lint (1.62.2)

var `histogramHandler` is unused

(unused)